src/CustomStyleCommand.ts

```diff
@@ -547,26 +554,49 @@ export function compareMarkWithStyle(
   if (style) {
     switch (mark.type.name) {
       case MARKSTRONG:
-        same = undefined !== style[STRONG];
+        if (undefined === mark.attrs[ATTR_OVERRIDDEN] || (undefined !== mark.attrs[ATTR_OVERRIDDEN] && !mark.attrs[ATTR_OVERRIDDEN])) {
```

Comment:
contents of this if should be a function to reduce duplication and clarify what the condition is.